09_qgep_dictionaries.sql

```diff
@@ -485,6 +485,8 @@ CREATE TABLE qgep_sys.dictionary_od_field (
  INSERT INTO qgep_sys.dictionary_od_field (class_id, attribute_id, table_name, field_name, field_name_en, field_name_de, field_name_fr, field_name_it, field_name_ro, field_description_en, field_description_de, field_description_fr, field_description_it, field_description_ro, field_mandatory, field_visible, field_datatype, field_unit_en, field_unit_description_en, field_unit_de, field_unit_description_de, field_unit_fr, field_unit_description_fr, field_unit_it, field_unit_description_it, field_unit_ro, field_unit_description_ro, field_max, field_min) VALUES (31,129,'reach','wall_roughness','wall_roughness','Wandrauhigkeit','K_COLEBROOK','zzz_Wandrauhigkeit','rugozitate_perete','yyy Hydraulische Kenngrösse zur Beschreibung der Beschaffenheit der Kanalwandung. Beiwert für die Formeln nach Prandtl-Colebrook (ks oder kb)','Hydraulische Kenngrösse zur Beschreibung der Beschaffenheit der Kanalwandung. Beiwert für die Formeln nach Prandtl-Colebrook (ks oder kb)','[mm]','[mm]','[mm]',ARRAY['Werkinformation']::qgep_od.plantype[],'true','decimal(5,2)','[mm]','roughness coefficient after Prandtl Colebrook (ks), unit milimeter [mm]','[mm]','Wandrauhigkeitsbeiwert nach Prandtl Colebrook (ks), Milimeter [mm]','','','','','','',0,100);
+ INSERT INTO qgep_sys.dictionary_od_field (class_id, attribute_id, table_name, field_name, field_name_en, field_name_de, field_name_fr, field_name_it, field_name_ro, field_description_en, field_description_de, field_description_fr, field_description_it, field_description_ro, field_mandatory, field_visible, field_datatype, field_unit_en, field_unit_description_en, field_unit_de, field_unit_description_de, field_unit_fr, field_unit_description_fr, field_unit_it, field_unit_description_it, field_unit_ro, field_unit_description_ro, field_max, field_min) VALUES (31,9999999,'reach','swmm_default_coefficient_of_friction','swmm_default_coefficient_of_friction',NULL,NULL,NULL,NULL,'Default coefficient of friction for SWMM export.',NULL,NULL,NULL,NULL,NULL,'true','smallint','[m^(1/3)/s]','Manning-Strickler [m^(1/3)/s]','[m^(1/3)/s]','Manning-Strickler [m^(1/3)/s]','','','','','','',0,999);
```

Comment:
To follow the same logic as with the attribute creation - then also move this to https://github.com/QGEP/datamodel/blob/master/10_dict_extension.sql

Reply:
ok, I agree - that was wrong with  https://github.com/QGEP/datamodel/blob/master/10_dict_extension.sql